lib/dune

```diff
@@ -68,7 +68,7 @@
  (name nec)
  (public_name ptt.nec)
  (modules nec)
- (libraries mirage-time mirage-clock mirage-random ptt ptt.map ptt.server
+ (libraries arc mirage-time mirage-clock mirage-random ptt ptt.map ptt.server
```

Comment:
once this is ready and merged, please remind me that I want to remove the mirage-clock and mirage-random dependencies ;) -- also mirage-time.